t/rest_server/.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,106 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: 'deploy'
+
+on:
+  workflow_dispatch:
+  workflow_call:
+
+concurrency:
+  group: '${{ github.workflow }}-${{ github.head_ref || github.ref }}'
+  cancel-in-progress: true
+
+env:
+  # This should be the only env variable defined in this file. Define all other
+  # variables in the bootstrap file.
+  # More environments are expected to be added in the future.
+  BOOTSTRAP_ENV_FILE: '.github/config/autopush.env'
```

Comment:
I imagine that more often than not, there will be more than one environment. It would be good to include at least 2 environments in this example. Can we add a production env just for demonstration sake?

Are you planning to have the environment name be an input value?

Reply:
Environment as input value, conditional logic demonstrated:
Autopush
https://github.com/drevell/abc-test-1/actions/runs/5523161977/jobs/10073697846#step:3:1
Prod
https://github.com/drevell/abc-test-1/actions/runs/5523171101/jobs/10073720473#step:4:1

Reply:
Remove production .env for now until we figure out a good variable substitution scheme